FakeItEasy.sln

```diff
@@ -10,13 +10,13 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		.mailmap = .mailmap
 		appveyor.yml = appveyor.yml
 		build.cmd = build.cmd
+		cacert.pem = cacert.pem
 		CONTRIBUTING.md = CONTRIBUTING.md
 		FakeItEasy.Dictionary.xml = FakeItEasy.Dictionary.xml
 		FakeItEasy.ruleset = FakeItEasy.ruleset
 		FakeItEasy.snk = FakeItEasy.snk
 		Gemfile = Gemfile
 		Gemfile.lock = Gemfile.lock
-		cacert.pem = cacert.pem

```

Comment:
I didn't intend to reorder those lines, but I guess it can't hurt, and there's no point if fighting VS over this.